makefile.shade

```diff
@@ -522,4 +593,12 @@ functions
                         .Replace("\n", "|n")
                         .Replace("]", "|]");
         }
+
+        string ProcessExcludesForPackages(string jsonEntry)
+        {
+            var newEntry = jsonEntry.Replace("Microsoft.Net.Http.Client\": \"1.0.0-*\"", "Microsoft.Net.Http.Client\": \"1.0.0-beta3-*\"");
+            newEntry = newEntry.Replace("Microsoft.AspNet.HttpFeature\": { \"version\": \"1.0.0-*\"", "Microsoft.AspNet.HttpFeature\": { \"version\": \"1.0.0-beta2-*\"");
```

Comment:
we have one project that is referencing HttpFeature hence I added it here. Let me find that and get back